### DISCUSSION TITLE
Howto change suricata.yaml settings

### DISCUSSION THREAD

**Discussion by Nippler1** [Security-Onion-Solutions/securityonion#7386] [Unsupported Versions] [answered]
Hello, everyone,

I installed 2.3.100 as a standalone security onion installation. Unfortunately have some lost packets at suricata.
I would like to work with the below Experiment a little with the Suricata parameters.
- cluster-type is set to cluster-flow with 15 threads (15 for Suricata, 15 for Zeek)
- The two sniffing interfaces are set to ethtool -L eno49/eno50 combined 1
- IRQ balancing is disabled
- Total incoming load on both interfaces approx. up to 1.3 Gb/s
- Suricatca loses about 4% packets on average 

I would like to customize the suricata.yaml. Unfortunately, only the parameters that are described in the document are available:

https://docs.securityonion.net/en/2.3/suricata.html?highlight=suricata#performance

How can I change other parameters? For example:
af-packet:
  ring-size:
  block size:
max-pending-packets:
stream:
  checksum-validation:
  memcap:
  hash size:

And so forth....
Is that even possible? If so, please provide a short howto 
Maybe someone else has an idea for tuning  ;)

Thanks very much.

**Comment by dougburks** [comment, marked as answer]
Under the Performance section is another section titled Configuration which tells you how to customize suricata.yaml:
https://docs.securityonion.net/en/2.3/suricata.html?#configuration
